js/src/admin/components/ExtensionPage.js

```diff
@@ -195,6 +196,21 @@ export default class ExtensionPage extends AdminPage {
       }
     });
 
+    const extension = this.extension;
+    items.add(
+      'readme',
+      Button.component(
+        {
+          icon: 'fab fa-readme',
+          class: 'Readme-link',
```

Comment:
Maybe `Button--link` could work here instead of styling it as a link manually? Haven't checked, though.